main.go

```diff
@@ -38,6 +31,13 @@ func main() {
 				os.Exit(143)
```

Comment:
os.Exit in a goroutine?

Reply:
I think you are right that it's not the best solution.

As far as I understand this behaviour is save, cause exit called only after waiting for a cleanup group and os.Exit as far as I understood termintates all routines.

What would you propose to improve this?

Reply:
@dm3ch The work is awesome. I will try to work on this issue. when this is ready. v0.152.0 will be released.